config.json

```diff
@@ -1,5 +1,6 @@
 {
-    "username":"",
-    "password":"",
-    "port": 3333
+    "Port": 3333,
```

Comment:
not a blocker, but is there a reason why we are capitalizing things?

Reply:
possibly a bad merge?

Reply:
also why is it not in alphabetical order anymore?

not really a problem now, but if there were 1234 things in the config it would be

Reply:
looks like definitely a bad merge.

It was not ever in any order.

Reply:
should be fixed now